README.md

```diff
@@ -1,5 +1,9 @@
 # SGX Experiments in Faasm
 
+This repository contains instructions to run SGX experiments on Faasm.
+For the moment, though, it contains instructions to provision an SGX-enabled
+VM and test that it can interact with the attestation service in Azure.
+
```

Comment:
I know we don't really enforce style on `.md` files but in the rest of the repos we put a linebreak before and after code snippets. Doesn't change the presentation of the rendered markdown but it makes the source easier to read.